.github/workflows/lint-test.yaml

```diff
@@ -129,9 +108,12 @@ jobs:
         uses: helm/kind-action@v1.10.0
         if: steps.list-changed.outputs.changed == 'true'
 
-      - name: Run chart-testing (install)
+      - name: Run chart-testing (install ${{ matrix.test_case }})
         id: install
         if: steps.list-changed.outputs.changed == 'true'
         run: |
-            ct install --target-branch ${{ github.event.repository.default_branch }} \
-            --helm-extra-set-args "--set=postgresql.enabled=true --set=postgresql.global.postgresql.auth.password=testing123456 --set=internalDatabase.enabled=false --set=externalDatabase.enabled=True --set=externalDatabase.type=postgresql --set=externalDatabase.password=testing123456"
+          if [[ ${{ matrix.test_case }} == 'Default - no custom values' ]]; then
+            ct install --target-branch ${{ github.event.repository.default_branch }}
```

Comment:
Just giving it an empty string should work, no? Then we don't need a conditional here.

Reply:
We shouldn't need a second PR to test this, just make a temporary commit that triggers the workflow to ensure it works.